modal/cli/volume.py

```diff
@@ -183,22 +183,20 @@ async def consumer():
     await asyncio.gather(*tasks)
 
 
-@volume_cli.command(
-    name="get",
-    help="""Download a file from a shared volume.
-
-Specifying a glob pattern (using any * or ** patterns) as the remote_path will download all matching *files*, preserving
-the source directory structure for the matched files.
-
-For example, to download an entire shared volume into `dump_volume`:
-
-> modal volume get <volume-name> "**" dump_volume
-
-Use "-" (a hyphen) as LOCAL_DESTINATION to write contents of file to stdout (only for non-glob paths)
-""",
-)
+@volume_cli.command(name="get")
 @synchronizer
 async def get(volume_name: str, remote_path: str, local_destination: str = typer.Argument("."), force: bool = False):
```

Comment:
Displays more nicely in `--help` and in our web docs. For example, see how https://modal.com/docs/reference/cli/app#modal-app-shell displays using this technique.

<img width="811" alt="image" src="https://user-images.githubusercontent.com/12058921/197929834-df9e5b56-2796-40b1-be5b-702323b1cefc.png">